src/sections/Projects/Sistent/components/tabs/guidance.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { navigate } from "gatsby";
+import { useLocation } from "@reach/router";
+import { Row } from "../../../../../reusecore/Layout";
+import { SistentThemeProvider } from "@layer5/sistent";
+import { SistentLayout } from "../../sistent-layout";
+
+import TabButton from "../../../../../reusecore/Button";
+import { useStyledDarkMode } from "../../../../../theme/app/useStyledDarkMode";
+
+const TabGuidance = () => {
+  const location = useLocation();
+  const { isDark } = useStyledDarkMode();
+
+  return (
+    <SistentLayout title="Tab">
+      <div className="content">
+        <a id="Identity">
+          <h2>Tab</h2>
+        </a>
+        <p>
+          Tabs allow users to switch between different views or sections within the same page without navigating away.
+        </p>
+        <div className="filterBtns">
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/tab"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/tab")}
```

Comment:
@Jaishree2310 There is a typo here, it redirects to an error page. 

It should be: /projects/sistent/components/tab**s**
```suggestion
            onClick={() => navigate("/projects/sistent/components/tabs")}
```